website/views.py

```diff
@@ -98,11 +97,6 @@ def _render_nodes(nodes, auth=None, show_path=False):
 
 @collect_auth
 def index(auth):
-    """Redirect to dashboard if user is logged in, else show homepage.
-
-    """
-    if auth.user:
-        return redirect(web_url_for('home'))
     return {}
```

Comment:
Flask's `url_for` [calls out](https://github.com/mitsuhiko/flask/blob/0.10.1/flask/helpers.py#L304-L305) to Werkzeug's [`build`](https://github.com/mitsuhiko/werkzeug/blob/0.10.4/werkzeug/routing.py#L1666-L1693).